src/nashpy/learning/replicator_dynamics.py

```diff
@@ -28,3 +28,54 @@ def replicator_dynamics(A, y0=None, timepoints=None):
 
     xs = odeint(func=get_derivative_of_fitness, y0=y0, t=timepoints, args=(A,))
     return xs
+
+
+def get_derivative_of_asymmetric_fitness(x, t, A, B):
+    """
+    Find the derivative of fitness function for the asymmetric replicator
+    dynamics scenario
+    """
+    separator = A.shape[0]
+    x1 = x[:separator]
+    x2 = x[separator:]
+
+    f1 = np.dot(A, x2)
+    f2 = np.dot(x1, B)
+
+    phi1 = np.dot(f1, x1)
+    phi2 = np.dot(f2, x2)
```

Comment:
I believe we don't have to use `np.dot` anymore:

```suggestion
    f1 = A @ x2
    f2 = x1 @ B

    phi1 = f1 @ x1
    phi2 = f2 @ x2
```